### PR TITLE
unit-tests-jest - Tev Nicolas

### PR DESCRIPTION
**Pull request by tevnicolas**
Screenshot 2024-04-12 at 12 30 55 PM

### CODE REVIEW

File: unit-tests-jest/numbers.test.ts [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    let result = toDollars(number);
+    expect(result).toBe('$55.00');
+    number = 55.01;
+    result = toDollars(number);
+    expect(result).toBe('$55.01');
+    number = 55.1;
+    result = toDollars(number);
+    expect(result).toBe('$55.10');
+  });
+  it('edge cases', () => {
```
Comment (RobertGardner) [inline unit-tests-jest/numbers.test.ts:28]: Best practice is to put each edge case in its own test.

File: unit-tests-jest/numbers.ts [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+/**
+ * Returns a number formatted in dollars and cents.
+ */
+export function toDollars(amount: number): string {
+  const rounded = Math.abs(amount).toFixed(2);
+  if (amount >= 0) {
+    return `$${rounded}`;
+  } else {
+    return `-$${rounded}`;
```
Comment (RobertGardner) [inline unit-tests-jest/numbers.ts:16]: Nice!

**Review by RobertGardner — APPROVED** [review]
